### PR TITLE
Add PlayMode tests to the PR and CI pipelines.

### PR DESCRIPTION
**Pull request by wiwei**
As we add more playmode tests we want to make sure that other folks aren't breaking them inadvertently. Easiest way of doing this is to enable these tests as part of PR validation (and CI as well)

### CODE REVIEW

**Comment by wiwei** [comment]
Hold off on reviewing this for now - Need to run this through the pipelines to see if they actually work or if playmode is weird for other reasons.

File: pipelines/pr.yaml [hunk 1]
```diff
@@ -153,7 +153,32 @@ steps:
...
+  displayName: 'Run EditMode tests'
+
+- powershell: |
+   $editor = Get-ChildItem ${Env:Unity2018.3.7f1} -Filter 'Unity.exe' -Recurse | Select-Object -First 1 -ExpandProperty FullName
+   
+   Write-Host "======================= PlayMode Tests ======================="
+   
+   $logFile = New-Item -Path .\playmode-test-run.log -ItemType File -Force
+   
+   $proc = Start-Process -FilePath "$editor" -ArgumentList "-projectPath $(Get-Location) -runTests -testPlatform playmodemode -batchmode -logFile $($logFile.Name) -editorTestsResultFile .\test-editmode-default.xml" -PassThru
```
Comment (keveleigh) [inline pipelines/pr.yaml]: Is this correct?

```suggestion
   $proc = Start-Process -FilePath "$editor" -ArgumentList "-projectPath $(Get-Location) -runTests -testPlatform playmode -batchmode -logFile $($logFile.Name) -editorTestsResultFile .\test-editmode-default.xml" -PassThru
```

Comment (wiwei) [reply pipelines/pr.yaml]: O_O

**Comment by wiwei** [comment]
Looks like the build machines obviously don't have permissions setup to enable microphone stuff - could actually possibly use this opportunity to not error out on the microphone/dictation providers.

Please hold off looking at this until I have addressed that.

**Comment by wiwei** [comment]
I'm going to close out this PR while I fix the other issue

**Pull request closed without merging**